Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "oranda"
 description = "🎁 generate beautiful landing pages for your projects"
 repository = "https://github.com/axodotdev/cargo-dist"
-version = "0.0.3"
+version = "0.0.4-prerelease.2"
```

Comment:
we should bring this back to oranda's version number (0.0.1)

Reply:
In the other PR since I still need the latest version of cargo dist, this merges to changelog PR